pkgs/applications/video/handbrake/default.nix

```diff
@@ -10,7 +10,7 @@
 { stdenv, lib, fetchFromGitHub,
   # Main build tools
   pkgconfig, autoconf, automake, libtool, m4, lzma, python3,
-  numactl,
+  numactl ? null,
```

Comment:
This serves no purpose as `numactl` is always defined: https://github.com/NixOS/nixpkgs/blob/e6ec27be9808ae6e661e500e9bc31f959ac449bc/pkgs/top-level/all-packages.nix#L18268